pyproject.toml

```diff
@@ -42,6 +42,8 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.7"
 typer = "*"
+aws-cdk-lib = "2.55.1"
+constructs = "^10.0.0"
```

Comment:
We can probably just use `*` as the version specifier for both of these, since we're not packaging anything for third party use - `poetry.lock` makes sure we're always using a specific version.